specification/_types/query_dsl/abstractions.ts

```diff
@@ -151,6 +151,7 @@ export class QueryContainer {
   terms?: TermsQuery
   terms_set?: SingleKeyDictionary<Field, TermsSetQuery>
   wildcard?: SingleKeyDictionary<Field, WildcardQuery>
+  wrapper: WrapperQuery
```

Comment:
This should be optional! 😱 
I'll send a pr to fix this.

Reply:
Yikes. You're absolutely right 😓